### ISSUE TITLE
Process immediately finishes without removing anything

### ISSUE THREAD

**Issue by stunlocked94** [apastel/ytmusic-deleter#42]
After installing the app and granting access to my youtube account i tried to get rid of all my liked music/my whole library.
When i click on "Delete All", or any other button for that matter, the process immediately finishes and nothing gets removed. I attached a screenshot.
![Screenshot 2024-03-28 095424](https://github.com/apastel/ytmusic-deleter/assets/73516723/8c4dee0e-9db8-453f-bcb6-0b864c53c4e7)
![Screenshot 2024-03-28 095649](https://github.com/apastel/ytmusic-deleter/assets/73516723/1f3a962a-4ca0-4b48-9f5c-8139d568c75c)

**Comment by apastel** [comment]
Huge thanks for bringing this to my attention. Someone else reported the same thing on Reddit but we both chalked it up to them running Windows on Mac using Parallels.

I'm also having an issue running the 2.1.0 version of the app but it actually prints an error message for me.

```
Showing confirmation dialog
Executing process: ytmusic-deleter ['-l', 'C:\\Users\\alexr\\AppData\\Roaming\\YTMusic Deleter', '-c', 'C:\\Users\\alexr\\AppData\\Roaming\\YTMusic Deleter', '-p', 'remove-library']
State changed: Starting
State changed: Running
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\alexr\dev\ytmusic-deleter\packages\app\target\YTMusic Deleter\_internal\ytmusic-deleter.exe\__main__.py", line 4, in <module>
ModuleNotFoundError: No module named 'packages.cli.cli'
```

Either way I will get this resolved ASAP and release a new version and you can let me know if it works for you. Hang tight!

**Comment by stunlocked94** [comment]
Sorry I forgot to mention: I am running this on Win11 with no pending updates, so I am assuming I am up to date. It's not throwing any errors for me, even on repeated runs. 

If I can provide more detailed logs (not sure where the program stores them) please let me know.

**Comment by apastel** [comment]
Good point, it should probably say where the logs are written to. It would be in `<User_folder>/AppData/Roaming/YTMusic Deleter`. (My user folder is C:\Users\alexr for example)

Usually the log that's displayed in the app console includes everything that's in the logs on the filesystem, so I'd be surprised if there's additional info in there but would be good to know if so. Thanks.

**Comment by apastel** [comment]
I just retested on another Windows machine and I got the exact same output as you. Nothing additional in the logs. I'll get this fixed ASAP!

**Comment by apastel** [comment]
I should mention that in the meantime if you don't want to wait or be my guinea pig for a new release, you can download the 1.6.0 version. https://github.com/apastel/ytmusic-deleter/releases/tag/1.6.0

**Comment by apastel** [comment]
@stunlocked94 Ok, I went ahead and re-published the 2.1.1 release. https://github.com/apastel/ytmusic-deleter/releases
It works now on my standalone Windows VM. Please give it a try and let me know what happens. Thanks.

**Comment by stunlocked94** [comment]
Looks great. Thank you for the quick fix. Everything works now.
